### ISSUE TITLE
Gulp watch should reload the browsers at any change in any CSS file.

### ISSUE THREAD

**Issue by LuismiBarcos** [liferay/liferay-js-themes-toolkit#235]
Creating a new theme I am using **gulp watch**, in particular for the CSS files. I note that if the CSS file is not at the same level of folders that the _custom.scss file, watch detect it but not reload the browser. If I want to reload the browsers I have to save the _custom.scss file.

Here are the steps to reproduce the issue:
1. Create a theme: `yo liferay-theme`
2. `gulp deploy`
3. Add a new folder
4. Inside the new folder, create a new .scss file and import it to _custom.scss
5. Start **gulp watch**
6. Make changes in the new file

In the log, we can see the following:
<img width="350" alt="imagen" src="https://user-images.githubusercontent.com/17163902/54674877-9c3dfc00-4afd-11e9-8277-3f6d93b0848a.png">

If I go to _custom.scss and press cmd+s (or ctrl+s) to save the file without make changes, I can see the following in the log:
<img width="495" alt="imagen" src="https://user-images.githubusercontent.com/17163902/54675551-16bb4b80-4aff-11e9-8d82-8ea4413b060c.png">

I think that if I make changes in any scss file, **watch should reload** the browsers.

**Comment by wincent** [comment]
@LuismiBarcos: if you want to test the fix before we release v8.0.1, you can pull the fixed `watch.js` file into your `node_modules` file like so:

```
# in your theme:
curl https://gist.githubusercontent.com/wincent/4a6253c552f4b8c43b545e32b9f64ad2/raw/fadc7a1ca56d4b13009b23bdb2db3a73b4fd8b84/gistfile1.txt > node_modules/liferay-theme-tasks/tasks/watch.js 
```

**Comment by wincent** [comment]
And v8.0.1 is now published to NPM (along with v9.0.0-alpha.1).
